internal/cmd/app.go

```diff
@@ -86,19 +86,26 @@ func GetApp(version string) *cli.App {
 				return err
 			}
 
+			if d.Config.State == "" {
```

Comment:
being safe, as we have several entrypoints, we have to make sure to set the default value for this if nothing is passed